src/content/en/2021/performance.md

```diff
@@ -1,20 +1,532 @@
 title: Performance
-description: TODO
-authors: []
-reviewers: []
-analysts: []
-editors: []
+description: Performance chapter of the 2021 Web Almanac covering Core Web Vitals (Largest Contentful Paint, Cumulative Layout Shift, First Input Delay) as well as First Contentful Paint and Time to First Byte
+authors: [siakaramalegos]
+reviewers: [rviscomi, kevinfarrugia, estelle, ziemek-bucko, jzyang, fili, samarpanda, edmondwwchan]
+analysts: [siakaramalegos, rviscomi, Nithanaroy]
+editors: [jzyang]
 translators: []
 results: https://docs.google.com/spreadsheets/d/13xhPx6o2Nowz_3b3_5ojiF_mY3Lhs25auBKM6eqGZmo/
+siakaramalegos_bio: Sia Karamalegos is a web developer, international conference speaker, and writer. She is a Google Developer Expert in Web Technologies, a Cloudinary Media Developer Expert, a Stripe Community Expert, and co-organizes the Eleventy Meetup. Check out her writing, speaking, and newsletter on <a hreflang="en" href="https://sia.codes/">sia.codes</a> or find her on <a hreflang="en" href="https://twitter.com/thegreengreek">Twitter</a>.
 featured_quote: TODO
 featured_stat_1: TODO
 featured_stat_label_1: TODO
 featured_stat_2: TODO
 featured_stat_label_2: TODO
 featured_stat_3: TODO
 featured_stat_label_3: TODO
+unedited: true
 ---
 
-## TODO
+## Introduction
+
+Performance is important for the user experience. Slow-to-load and slow-to-respond websites frustrate users and cause lost conversions. This is the first year that the <a hreflang="en" href="https://web.dev/vitals/">Core Web Vitals</a> have contributed to Google search rankings. As such, we've seen greater interest in improving website performance which is great news for users.
+
+<i>What are our top takeaways from this year's report?</i> First, we still have a long way to go in providing a good user experience. For example, faster networks and devices have not yet reached the point where we can ignore how much JavaScript we deliver to a site; and, we may never get there. Second, sometimes we misuse new features for performance, resulting in poorer performance. Third, we need better metrics for measuring interactivity, and those are on the way. And fourth, CMS- and framework-level work on performance can significantly impact user experience for the top 10M websites.
+
+<i>What's new this year?</i> We're excited to share performance data by traffic ranking for the first time. We also have all the core performance metrics from previous years. Finally, we added a deeper dive into the Largest Contentful Paint (LCP) element.
+
+### Notes on Methodology
+
+One thing that makes the performance chapter different from the others is that we rely heavily on the <a hreflang="en" href="https://developers.google.com/web/tools/chrome-user-experience-report">Chrome User Experience Report</a> (CrUX) for our analyses. Why? If our number one priority is user experience, then the best way to measure performance is with real user data (real user metrics, or RUM for short).
```

Comment:
For the first edit - I think of it as present continuous because it's always the case. Previous years, this year, and future years will rely heavily on it.

For questions-answer style edit... My personal style is to ask questions I think the reader is asking themselves. Kind of like this - https://bloggingwizard.com/improve-your-writing-style/

Just a way to make reading it more engaging.

Reply:
OK.  For me the rhythm of the read was off because you don't generally have 1 word sentences in the majority of your writing.  This makes sense because this is not a blog; a lot of what you're presenting is based on numbers and facts and your experience in the industry.  Which was why I was saying that it has a more professional writing style for the majority.  It was off-putting to be jumping between these different rhythms in your writing: Between the formal and the super casual.  At least, that was the case for me.

Reply:
But, again, you got to do what feels right to you.  :) Just go with what makes you feel good, I'm only making suggestions.